app/scripts/platforms/extension.js

```diff
@@ -31,6 +32,69 @@ class ExtensionPlatform {
       cb(e)
     }
   }
+
+  showTransactionNotification (txMeta) {
+
+    const status = txMeta.status
+    if (status === 'confirmed') {
+      this._showConfirmedTransaction(txMeta)
+    } else if (status === 'failed') {
+      this._showFailedTransaction(txMeta)
+    } else if (status === 'dropped') {
+      this._showDroppedTransaction(txMeta)
```

Comment:
I was already on the fence about showing dropped txs, but if Firefox risks not showing any, I think I'd rather remove the dropped handler.